platformcomponents/win-hc/button.json

```diff
@@ -92,6 +92,20 @@
       "#pressed": {
         "text": "@theme-text-primary-inverted"
       }
+    },
+    "dialpad": {
```

Comment:
Text colour needs to inverted when hovered or pressed in High Contrast.
![image](https://user-images.githubusercontent.com/27616697/140055124-42d2b323-b893-4ed7-ae51-039c98ea5ce0.png)